nodemailer/nodemailer-types.d.ts

```diff
@@ -1,149 +0,0 @@
-﻿// Type definitions for Nodemailer 1.3.2
```

Comment:
I combined this file with nodemailer.d.ts.  Typings needed this to all be in one file.